ENGDOCS-2220 (#20882)
docker/docker.github.io: content/manuals/desktop/faqs/linuxfaqs.md

Docker Desktop for Linux uses [VirtioFS](https://virtio-fs.gitlab.io/) as the
default (and currently only) mechanism to enable file sharing between the host
and Docker Desktop VM. 

> > [!IMPORTANT]

Reviewer: Could we have this, instead of the note:
The whole ID mapping information below included in a collapsable box, titled something like ""Additional information for Docker Desktop 4.34 and older", similar to what we have for example [here](https://docs.docker.com/desktop/install/windows-install/#:~:text=VDI%20environment.-,How,-do%20I%20switch). 
This way  users on 4.35+ can ignore the Additional info box as it is collapsed by default, and osers on older versions can expand it to read about it.

>

> > For Docker Desktop version 4.35 and later, you can ignore the manual setup steps below as this now happens automatically. 

Reviewer 2: ```suggestion
> For Docker Desktop version 4.35 and later, you can ignore the manual setup steps below as this is not required anymore. 
```
